application/common/components/Emailer.php

```diff
@@ -260,7 +265,25 @@ public function shouldSendInviteMessageTo($user, $isNewUser)
     public function shouldSendPasswordChangedMessageTo($user, $changedAttributes)
```

Comment:
Sorry - this wasn't a PR change, but I was wondering why $user is still a parameter if it's not being used in the function.
That applies to the next function also.

Reply:
Ohhh... good catch. That was partially a copy-paste oversight (for this function), but a logic oversight for the next function. It should also be checking whether the user has already received a welcome message.

Reply:
On further thought, since there is no supported way for a user to stop having a password (which is what it would take to allow them to later re-receive the welcome email), the extra code for preventing sending a welcome message to someone who has already received one seems unnecessary. If a user were somehow deleted from the system (not currently supported), and then were later re-added, re-sending them the welcome message doesn't seem like a bad thing.